Dockerfile

```diff
@@ -11,8 +11,10 @@
 
 WORKDIR /app
 
+
 ENV PYTHONUNBUFFERED=1
 
+
 COPY --from=builder /app .
```

Comment:
Ensure that the `COPY` command only includes necessary files for running the application. You might want to exclude files that are only needed during the build stage.